packages/react/src/publication/useEncryptedPublication.ts

```diff
@@ -75,7 +75,9 @@ export function useEncryptedPublication<T extends ContentPublication>({
         );
       }
 
-      const client = createGatedClient({
+      const createGatedClientFn = encryption.createGatedClient || createGatedClient;
```

Comment:
wondering if we could make the `encryption.createGatedClient` default to `createGatedClient` to avoid branching logic here?